manual/content/point-data/point-data.md

```diff
@@ -0,0 +1,349 @@
+
+The list of available path tokens and their corresponding locations are found in the openspace.cfg file.
+:::
+
+## Controlling the Point Size
+
+At the core, the size of the points is computed based on one parameter: a logarithmic exponent that decides the absolute size of the point. The exponential component exists to allow creating point clouds over very different scales and distances and should be set to match the scale of the dataset. It is used to compute the actual world scale size of the points, at their position in the 3D scene. For example, an exponent of 3 will lead to points with a size in the order of 1000 meters, while an exponent of 10 will lead to points with a size of about 10^10 = 100'000'000 meters.
+
+If not included in the asset, a default exponent is computed based on the positional information in the dataset. However, this should be seen as a starting point and you will likely want to modify it so that the scale of the points looks good based on the density, number of points and the spread of your particular dataset, as well as the use case for which it is to be shown.
+
+Secondly, a multiplicative factor that can be used to increase or decrease the *visual* size of the points also exists. This factor can be seen as a tool to quickly increase or decrease the appeared size of all points equally. This factor is applied at the last step, after the exponential scaling and any pixel size effects.
+
+To update the scaling in the points, add a `SizeSettings` table to your asset specification. See example:
+
+```lua
+  ...
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("path/to/dataset.csv"),
+    SizeSettings = {
+      -- Control the world-scale size of the points, here in the order of 10^15
+      ScaleExponent = 15.0,
+      -- Visually make all the points twice as large
+      ScaleFactor = 2.0
+    }
+  },
+  ...
+```
+
+:::{note}
+Since the exponent affects the physical, world-scale, size of the points, this means that points that are closer to the camera will appear larger than those that are further away, and increasing the exponent also enhances this effect. In contrast, increasing the multiplicative factor will increase the visual size of the points equally for all points. This is because it is applied in the very last step, after the exponential scaling and any pixel size effects.
+:::
+
+### Limit by Size in Pixels
+In addition to the world-size scale, it is also possible to limit the screen-space size of the points to a maximum size in pixels. This prevents the points from growing larger than the specified pixel size when the camera is approaching the dataset. An example is shown in the images below.
```

Comment:
Would you say that this is a recommended practice?

Reply:
Good question.  We do it almost everywhere for our own datasets (or rather, the Digital Universe datasets), to prevent the points from taking up too much screen-space real estate. 

However, I feel hesitant to recommend it before we have a solution that also works in a dome, for example.  But that might just be my own discomfort in recommending to use something that does not work to a 100%. For the users with planar displays, I guess it would be recommended.